lambda/writePass/index.js

```diff
@@ -72,6 +76,63 @@ exports.handler = async (event, context) => {
       numberOfGuests = 1;
     }
 
+    // Get current time vs booking time information
+    const today = new Date();
+    const currentDay = parseInt(today.toLocaleString('en-US', { day: '2-digit', timeZone: 'America/Vancouver' }));
```

Comment:
I added the date-fns `utcToZonedTime` helper, so you can do
```
const { utcToZonedTime } = require('date-fns-tz');

...

const localDate = utcToZonedTime(Date.now(), 'America/Vancouver');
const currentDay = localDate.getDate();
const currentHour = localDate.getHour();
```
or similar. I think it doesn't change it functionally though, just FYI

Reply:
Much cleaner solution. Implemented in `9408cb3`